languages/translations/hu.json

```diff
@@ -0,0 +1,493 @@
+                "name": "Forduljon {modifier} és maradjon {way_name:article} szakaszon",
+                "destination": "Forduljon {modifier} {destination:article} irányába",
+                "exit": "Forduljon be {modifier} {way_name:article} irányába"
+            },
+            "straight": {
+                "default": "Hajtson tovább egyenesen",
+                "name": "Hajtson tovább egyenesen {way_name:article} szakaszon",
+                "destination": "Hajtson tovább {destination} irányába",
+                "distance": "Hajtson tovább egyenesen {distance} távolságig",
+                "namedistance": "Hajtson tovább {way_name:article} szakaszon {distance} távolságig"
+            },
+            "sharp left": {
+                "default": "Forduljon élesen balra",
+                "name": "Forduljon élesen balra, majd maradjon {way_name:article} szakaszon",
+                "destination": "Forduljon élesen balra {destination:article} irányába"
+            },
+            "sharp right": {
+                "default": "Forduljon élesen jobbra",
+                "name": "Forduljon élesen jobbra, majd maradjon {way_name:article} szakaszon",
+                "destination": "Forduljon élesen balra {destination:article} irányába"
+            },
+            "slight left": {
+                "default": "Forduljon enyhén balra",
+                "name": "Forduljon enyhén balra, majd maradjon {way_name:article} szakaszon",
+                "destination": "Forduljon enyhén balra {destination:article} irányába"
+            },
+            "slight right": {
+                "default": "Forduljon enyhén jobbra",
+                "name": "Forduljon enyhén jobbra, majd maradjon {way_name:article} szakaszon",
+                "destination": "Forduljon enyhén jobbra {destination:article} irányába"
+            },
+            "uturn": {
+                "default": "Forduljon vissza",
+                "name": "Forduljon vissza, majd folytassa {way_name:article} szakaszon",
+                "destination": "Forduljon vissza {destination:article} irányába"
+            }
+        },
+        "depart": {
+            "default": {
+                "default": "Hajtson {direction:article} irányába",
+                "name": "Hajtson {direction:article} irányába {way_name:article} szakaszon",
+                "namedistance": "Hajtson {direction:article} irányába {way_name:article} felé {distance} távolságig"
+            }
+        },
+        "end of road": {
+            "default": {
+                "default": "Forduljon {modifier}",
+                "name": "Forduljon be {modifier} {way_name:article} irányába",
+                "destination": "Forduljon {modifier} {destination:article} felé"
+            },
+            "straight": {
+                "default": "Hajtson tovább egyenesen",
+                "name": "Hajtson tovább egyenesen {way_name:article} felé",
+                "destination": "Hajtson tovább egyenesen {destination} irányba"
```

Comment:
@kiskunk, maybe there should be `irányába` not `irányba`? And article `a(z)` is missing here?